core/domain/exp_services.py

```diff
@@ -250,13 +318,23 @@ def get_multiple_explorations_by_id(exp_ids, strict=True):
 
 
 def get_new_exploration_id():
-    """Returns a new exploration id."""
+    """Returns a new exploration id.
+
+    Returns:
+        str. A new exploration id.
+    """
     return exp_models.ExplorationModel.get_new_id('')
 
 
 def is_exp_summary_editable(exp_summary, user_id=None):
-    """Checks if a given user may edit an exploration by checking
-    the given domain object.
+    """Checks if a given user has permissions to edit the exploration.
+
+    Args:
+        exp_summary: ExplorationSummary. An ExplorationSummary domain object.
+        user_id: str. ID of the user whose permissions are being checked.
```

Comment:
I noticed that you used 'ID' in some places an 'id' in others, can you change them all to just 'id' for consistency? Thanks!